servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/SingleAddressGrpcClientBuilder.java

```diff
@@ -95,6 +96,14 @@
      */
     GrpcClientSecurityConfigurator<U, R> secure();
 
+    /**
+     * Updates the automatic retry strategy for the clients generated by this builder.
```

Comment:
Consider provide more motivation/context for this type of retries. We can clarify a few more things in javadoc about these auro retries:
1. designed for framework internals that happen before a request touches network;
2. will be handled as a first filter => other users filters won't see failures that were successfully retried;
3. are protocol-agnostic and do not have request meta;

Should we clarify that other retries should be handled by custom `StreamingHttpClientFilter`s?

Reply:
Good idea, lemme add some information without leaking details about the implementation.